RecoBTag/TensorFlow/test/plot.py

```diff
@@ -0,0 +1,73 @@
+import ROOT
```

Comment:
this file apparently plots only for deepDoubleX {B,C}. Please make it clear by renaming the file.